src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -129,7 +134,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         try {
             $server = new WebServer();
-            $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
+            $config = new WebServerConfig(
+                $documentRoot,
+                $env,
+                $input->getArgument('addressport'),
+                $input->getOption('router'),
+                $input->getOption('executable')
```

Comment:
Is in your oppinion "executable" correct name for what it holds? (e.g.: `php` or `php -c /path/to/ini` or `/usr/bin/custom_php`). Other options I considered were "command" (although that clashes with Symfony command) or "binary" (which is usually the file itself, not the whole command).